DigitalLearningSolutions.Data.Tests/Helpers/EmailTestHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Helpers
+{
+    using DigitalLearningSolutions.Data.Models.Email;
+    using MimeKit;
+
+    public static class EmailTestHelper
+    {
+        public const string DefaultHtmlBody = "<body style= 'font - family: Calibri; font - size: small;'>\r\n" +
+                                              "                                    <p>Test Body</p>\r\n" +
```

Comment:
Is the big spacing needed between `"` and `<p>` ?

Reply:
It seems that the spacing needs to exactly match what is in the string. The actual string in the GetDefaultEmail method that this is compared against was copied over, formatting intact, from the previous email sending methods, which had those large gaps in. Would we prefer a slightly more sensible sensible layout?